File: CSharp/File & Function Length/workspace/file_function_length_benchmark/cases/NestedControlDemo.cs
namespace Cases;

public class NestedControlDemo
{
    public int NestedExample(int x)
    {
        if (x > 0)
        {
            if (x > 1)
            {
                if (x > 2)
                {
                    return x;
                }
            }
        }
        return 0;
    }
}

File: Java/Parameter Count/workspace/parameter_count_benchmark/src/cases/NestedControlDemo.java
package cases;

public class NestedControlDemo {
    public int nestedExample(int x) {
        if (x > 0) {
            if (x > 1) {
                if (x > 2) {
                    return x;
                }
            }
        }
        return 0;
    }
}
